devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy

```diff
@@ -29,7 +29,7 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF, 0019", outClusters: "0019", deviceId: "0100", manufacturer: "AduroSmart Eria", model: "ZLL-DimmableLight", deviceJoinName: "Eria ZLL Dimmable Bulb"
 
 		// IKEA
-		fingerprint inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E26 opal 1000lm", deviceJoinName: "IKEA TRÅDFRI LED Bulb"
```

Comment:
This is a valid model name and this fingerprint should not be removed

Reply:
Please excuse me, it has looked like a typo without the W. Then could you please add the E26 version of the W opal as a few lines below the E27 is.
I am really sorry for the confusion.
Thanks!

Reply:
You can go ahead and update the PR to add the new fingerprint and leave this one in place

Reply:
Updated as discussed. Kept original fingerprint and added E26 just below E27.